domains/transfer-agreements/TransferAgreements.API/API/Controllers/TransferAgreementHistoryEntriesController.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using API.ApiModels.Responses;
+using API.Data;
+using API.Extensions;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Controllers;
+
+[Authorize]
+[ApiController]
+[Route("api/history/transfer-agreements")]
```

Comment:
With this route, you also need to update the ingress routes in eo-base-environment

Reply:
Noted